src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs

```diff
@@ -197,6 +193,92 @@ public void LogSqlDependency_SinksToApplicationInsights_ResultsInSqlDependencyTe
             }
         }
 
+        [Fact]
+        public void LogEventWithComponentName_SinksToApplicationInsights_ResultsInTelemetryWithComponentName()
+        {
+            // Arrange
+            string componentName = _bogusGenerator.Commerce.ProductName();
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory(config => config.Enrich.WithComponentName(componentName)))
```

Comment:
Can you add a test for component name with a dependency as well please?

This to verify that the application map shows a node with that name

Reply:
Looks like this was added

Reply:
Added...
Is this what you want:
![image](https://user-images.githubusercontent.com/9039753/78527862-4e666580-77de-11ea-9cf7-47916a9c5827.png)
![image](https://user-images.githubusercontent.com/9039753/78527868-545c4680-77de-11ea-9118-67bfdebc6813.png)